### PR TITLE
Mobile navigation

### PR DESCRIPTION
**Pull request by viru2001**
Added mobile navigation 

mobile nav

### CODE REVIEW

File: src/frontend/components/Navbar/HamburgerNav.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const HamburgerNav = ({
+  status,
+  wishlist,
+  signOutHandler,
+  setAuth,
+  username,
+  cart,
+}) => {
+  return (
+    <section id="hamburger-nav-menu" class="d-none flex-wrap my-8">
```
Comment (ravikumar0403) [inline src/frontend/components/Navbar/HamburgerNav.jsx]: change `class` to `className` here

File: src/frontend/components/Navbar/NavItem.jsx [hunk 2]
```diff
@@ -9,10 +9,13 @@ const NavItem = ({ navItemTitle, number, icon, navigateTo }) => {
...
+            className="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute"
+            style={number > 9 ? { padding: "1px 4px" } : { padding: "1px 8px" }}
+          >
             {number}
           </span>
-        ) }
+        )}
       </span>
       <span className="nav-link-description text-sm">{navItemTitle}</span>
     </Link>
```
Comment (ravikumar0403) [inline src/frontend/components/Navbar/NavItem.jsx:23]: Since this is a navbar, you can use the NavLink component

**Review by ravikumar0403 — APPROVED** [review]
Looks good,

File: src/frontend/components/Navbar/NavItem.jsx [hunk 3]
```diff
@@ -9,10 +9,13 @@ const NavItem = ({ navItemTitle, number, icon, navigateTo }) => {
       <span className="p-relative">
         <i className={`fas fa-${icon} text-md`}></i>
         {number !== undefined && (
-          <span className="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute">
+          <span
+            className="badge icon-badge-position text-sm font-wt-bold rounded-full p-absolute"
+            style={number > 9 ? { padding: "1px 4px" } : { padding: "1px 8px" }}
```
Comment (SRUJANA-PENUGONDA13) [inline src/frontend/components/Navbar/NavItem.jsx]: Give some generic name to 'number' like item's count. When I see that variable I didn't get immediate idea on what it refers to.

File: src/frontend/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -18,58 +19,77 @@ const Navbar = () => {
...
+        </Link>
+        <button
+          id="hamburger-btn"
+          className="btn text-md ml-auto mr-8 p-4"
+          onClick={() => setIsMobileNavOpen(!isMobileNavOpen)}
+        >
+          <i className="fas fa-bars hamburger-icon"></i>
+        </button>
+        <section id="nav-menu-without-hamburger" className="d-flex m-2">
+          <nav className="nav-links d-flex align-center">
```
Comment (SRUJANA-PENUGONDA13) [inline src/frontend/components/Navbar/Navbar.jsx:41]: I have observed you have used same nav code in HamburgerNav.jsx. If possible create another component and reuse it.

**Review by SRUJANA-PENUGONDA13 — COMMENTED** [review]
Over all looks good to me. Added few suggestions once look into it

File: src/frontend/components/Navbar/Navbar.jsx (hunk 4 above)
Comment (viru2001) [reply src/frontend/components/Navbar/Navbar.jsx:41]: will think about it

File: src/frontend/components/Navbar/NavItem.jsx (hunk 3 above)
Comment (viru2001) [reply src/frontend/components/Navbar/NavItem.jsx]: done

**Pull request merged**